src/public-lib/lib/persistentApiClientCache.spec.ts

```diff
@@ -0,0 +1,222 @@
+import { assert } from 'chai';
+import sinon from 'sinon';
+import { cacheGetMethods, clearPersistentCache } from './persistentApiClientCache';
+import { sleep } from './../../shared/lib/TimeUtils'; // normal file path was causing buildPublicLib to break?
```

Comment:
@alisman Any idea why I had to do this? Previously it was `shared/lib/TimeUtils`, and it built fine, but `yarn run buildPublicLib` failed.

Reply:
sorry i missed this.  my guess is that it has to do with aliases.    in main project we have alias set up for "shared" that resolve to a to real path.    Probably there's no such alias set up in the public lib config and so it doesn't work in that context.  just a guess though! (@Onur?)

Reply:
Okay, I took some time and figured this out. I added the shared alias for the public lib, since that dependency should be acceptable. I also realized that I was breaking the public lib build with the AppConfig import for related reasons. That import doesn't belong in the public lib, so I pulled the check out into `config.ts` and now everything builds